compute_period: 20        # ms
max_speed_linear: 1.0     # m/s
max_speed_angular: 2.0    # rad/s